src/flashbar/collapsible-flashbar.tsx

```diff
@@ -322,6 +328,13 @@ export default function CollapsibleFlashbar({ items, ...restProps }: FlashbarPro
           )}
           onClick={toggleCollapseExpand}
           ref={notificationBarRef}
+          {...getAnalyticsMetadataAttribute({
```

Comment:
1. This one doesn't get any analytics data
2. Why this one is not using `getComponentsAnalyticsMetadata`?